### ISSUE TITLE
Implement GraphLinesRenderer

### ISSUE THREAD

**Issue by jsbean** [jsbean/DNM#210]
Abstract this process out of `GraphLayer`.

Danger: parallel inheritance hierarchies? But is this unavoidable?

**Comment by jsbean** [comment]
```Swift
struct GraphLinesRenderer {
    var graph: Graph
    var amountLines: Int // currently spaced, for now
    var staffHeight: StaffHeight
    var linePoints: [GraphLinePoint]
    init() { ... }
}
```

**Comment by jsbean** [comment]
Closed with #221
